src/js-v2-api-specification.md

```diff
@@ -0,0 +1,309 @@
+
+Live
+
+---
+
+#### goToPage Function <a name="goToPage"></a>
+
+`nhsapp.navigation.goToPage(appPage: string enum)`
+
+##### Description
+Redirects the user to a page in the NHS App specified by the appPage string enumeration.
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| appPage                 | **Type:** nhsapp.navigation.AppPage *Required*<br>The page to navigate to.                                                                                                     |
+
+##### AppPage enumerations
+
+| Values         |                        |
+|----------------|------------------------|
+| HOME_PAGE      | NHS App Home Page      |
+| SERVICES       | Services Hub Page      |
+| YOUR_HEALTH    | Your Health Hub Page   |
+| MESSAGES       | Messages Hub Page      |
+| UPLIFT         | P9 Uplift Page         |
+
+
+##### Example Usage
+
+```js
+nhsapp.navigation.goToPage(nhsapp.navigation.AppPage.MESSAGES)
+```
+
+##### Status
+
+Live
+
+---
+
+#### openBrowserOverlay Function <a name="openBrowserOverlay"></a>
+
+`nhsapp.navigation.openBrowserOverlay(overlayUri: Uri)`
+
+##### Description
+
+Redirects the user to a specified page that will open in a browser overlay view.
+
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| overlayUri                 | **Type:** Uri *Required*<br>The page to open in the browser overlay.                                                                                                     |
+
+##### Example Usage
+
+```js
+nhsapp.navigation.openBrowserOverlay("https://www.nhs.uk/")
+```
+
+##### Status
+
+Live
+
+---
+
+#### openExternalBrowser Function <a name="openExternalBrowser"></a>
+
+`nhsapp.navigation.openExternalBrowser(browserUri: Uri)`
+
+##### Description
+
+Redirects the user to a specified page that will open in the device's default browser application.
+
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Not sure how much it matters, but whitespace alignment.

Reply:
fixed